processes/development/new-app-setup/README.md

```diff
@@ -55,32 +55,35 @@ This assumes the discovery week has already happened.
 
 ## CI
 
-AWS config:
+#### AWS config:
 
 1. Create IAM user with read-only access to production S3 bucket and read/write access  to staging S3 bucket
     1. [Add policy](https://drive.google.com/open?id=0B4JoTt-NyIq5Y2RuYjZPTFAwd0U)
     2. Ensure the user has API credentials, no password
 
-MongoDB create users:
+#### MongoDB create users:
 
 1. Create read-only user for production DB
 2. Create user on staging DB (not read-only)
 
-Semaphore config:
+#### Semaphore config:
 
 1. Create new project
     1. NOTE: Paul needs to do this step currently. Will fix...
-1. Edit build settings:
+1. Edit Build Settings:
     1. Go to [https://semaphoreci.com/okgrow/](https://semaphoreci.com/okgrow/)&lt;PROJECT_NAME&gt;/settings
     1. Set the Node version to node.js 6.3.1
     1. Add each of these lines under setup
         1. `curl https://install.meteor.com/ | sh`
         1. `meteor --version`
-        1. `npm install`
-        1. `cp settings.json.example settings.json`
+        1. `meteor npm install`
+        1. `meteor npm start` _wip: find and add instructions to listen for success and exit process_
```

Comment:
@rdickert I'll play around with it in SSH.  If anything, I think I should be able run the chimp smoke test.  But I think the test structure needs to be fixed (it's breaking with the unit&integration tests)